src/main/java/org/sagebionetworks/web/client/widget/FullWidthAlert.java

```diff
@@ -0,0 +1,145 @@
+package org.sagebionetworks.web.client.widget;
+
+import org.gwtbootstrap3.client.ui.Alert;
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.Icon;
+import org.gwtbootstrap3.client.ui.constants.AlertType;
+import org.gwtbootstrap3.client.ui.constants.ButtonType;
+import org.gwtbootstrap3.client.ui.constants.IconType;
+import org.gwtbootstrap3.client.ui.html.Span;
+import org.gwtbootstrap3.client.ui.html.Strong;
+
+import com.google.gwt.core.shared.GWT;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.Window;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+
+public class FullWidthAlert implements IsWidget {
+	public interface Binder extends UiBinder<Alert, FullWidthAlert> {
+	}
+
+	private static Binder uiBinder = GWT.create(Binder.class);
+
+	@UiField
+	Icon icon;
+	@UiField
+	Span messageSpan;
+	@UiField
+	Strong messageTitle;
+	@UiField
+	Button primaryButton;
+	@UiField
+	Button secondaryButton;
```

Comment:
Added the ability to configure a secondary button (buttons are not shown unless you set the button text)